src/formatDate.js

```diff
@@ -9,6 +9,46 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const splittedDate = date.split(fromFormat[3]);
```

Comment:
Consider destructuring the `fromFormat` array to improve readability. For example, `const [yearKey, monthKey, dayKey, separator] = fromFormat;` allows you to use meaningful variable names instead of array indices.

Reply:
I'm not sure that fromFormat will always have the same filling, that the year is always in first place, month in second and date in third